simulation/__main__.py

```diff
@@ -146,6 +146,13 @@ def format_uuid(str):
         help="Include new build projections (from constants.py). Installers will also build heat pumps in new builds from 2025.",
     )
 
+    parser.add_argument(
+        "--heat-pump-awareness-intervention-factor",
+        type=float_between_0_and_1,
+        default=0,
+        help="A value between 0 and 1 which determines how quickly heat pump awareness increases over time with the heat pump awareness intervention. A value of 0 indicates no increase in heat pump awareness.",
```

Comment:
Just to check: is 0 'no increase' or 0 = 'no awareness'?

Reply:
No increase in awareness from the initial value. I will clarify in description.